docs/catalog/cloud/tag-resources.md

```diff
@@ -0,0 +1,34 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- operations
+- compute
+- kubernetes
+- role:software-engineer
+- size:small
+---
+
+# Tag resources
+
+## Description
+Getting the right information and insights at the right time is important for producing reports about performance and resource utilization.
+
+## Solution
+Set tags on your cloud resources at the infrastructure level, to enable monitoring of your applications overall performance.
+
+
+## SCI Impact
```

Comment:
If the pattern doesn't impact the SCI score in some way, then it very likely isn't a pattern we would support. However, this seems that it would align to the [Measurement](https://learn.greensoftware.foundation/practitioner/measurement) principle so I'd like to get @greenhsu123's thoughts on this for inclusion.